srcs/utils/error_exit.c

```diff
@@ -20,6 +24,11 @@ void	error_message(t_error code)
 		ft_putstr_fd("Wrong number of arguments.\n", STDERR_FILENO);
 	else if (code == NO_OLDPWD)
 		ft_putstr_fd("OLDPWD not set.\n", STDERR_FILENO);
+	/* can you check the input:     > test       to see if you have leaks or a segfault ? */

```

Comment:
why is this here, isn't this related to the seg fault when there's no command? Or is this something else?

Reply:
oh ! I actually corrected that. I forgot to get rid of the comment.